buildin/module.js

```diff
@@ -4,16 +4,21 @@ module.exports = function(module) {
 		module.paths = [];
 		// module.parent = undefined by default
 		if(!module.children) module.children = [];
-		Object.defineProperty(module, "loaded", {
-			enumerable: true,
-			configurable: false,
-			get: function() { return module.l; }
-		});
-		Object.defineProperty(module, "id", {
-			enumerable: true,
-			configurable: false,
-			get: function() { return module.i; }
-		});
+		try{
+      Object.defineProperty(module, "loaded", {
+        enumerable: true,
+        configurable: false,
+        get: function() { return module.l; }
+      });
+      Object.defineProperty(module, "id", {
+        enumerable: true,
+        configurable: false,
+        get: function() { return module.i; }
+      });
+		}catch (e){
+      module["loaded"] = function() { return module.l; };
+      module["id"] = function() { return module.i; };
```

Comment:
This changes the type of `module["loaded"]` and `module["id"]` from `typeof module.l` / `typeof module.i` to `function`.